README.md

```diff
@@ -49,12 +54,13 @@ cd ~/projects
 ### 🐋 Docker
 
 ```console
-docker run --interactive --tty --rm --volume $PWD/.dumps:/app/.dumps --volume $PWD/.logs:/app/.logs --volume $PWD:/data 20tab/talos-django:latest
+docker run --interactive --tty --rm --volume $PWD/.talos_dumps:/app/.dumps --volume $PWD/.talos_logs:/app/.logs --volume $PWD:/data 20tab/talos-django:latest
```

Comment:
I would reuse the same dirs locally so as to keep the dumps and logs in "buckets" the user would expect to find such data.

Reply:
In the `~/projects` directory where it is suggested to execute the setup with `docker run` there is no context related to the execution of talos. When mounting the two volumes, two unintelligible hidden directories `~/projects/.logs` and `~/projects/.dumps` are generated. 
I consider a result like `~/projects/.talos_logs` and `~/projects/.talos_dumps` to be clearer and more understandable.

Reply:
Moreover, that would be more garbage added to my system. I really do not like that. :D 

Moreover:

Why unintelligible? Mappings to `.dumps` and `.logs` are in the command itself, the user cannot get mistaken in any way.

`talos_dumps` and `talos_logs` are hidden too, so that's not a valid argument either.